cli/args/flags.rs

```diff
@@ -2747,10 +2747,11 @@ Note: running multiple `deno test --clean` calls in series or parallel for the s
         .value_hint(ValueHint::AnyPath),
     )
     .arg(
-      watch_arg(false)
+      watch_arg(true)
         .conflicts_with("no-run")
         .conflicts_with("coverage"),
     )
+    .arg(hmr_arg(true))
```

Comment:
HMR does not work with `deno test`, because tests are only executed once. Please undo this.

Reply:
Done, I also had to add a check for `watch_arg_parse_with_paths` method, to make sure the HMR argument exists for the command. Otherwise, it panicked
![image](https://github.com/user-attachments/assets/e9a69ccd-070e-403d-a722-7d5d9341be16)
![image](https://github.com/user-attachments/assets/7b975aad-d950-40a4-ac5b-e0693a38bdf3)